### PR TITLE
Add WGSL strict LALR(1) check to CI

### PR DESCRIPTION
**Pull request by dneto0**
Adds the 'wgsl.lalr.txt' target to 'all in wgsl/Makefile
Adds ~7s runtime on a moderate laptop.

Refactor the wgsl/Makefile somewhat

### CODE REVIEW

**Comment by dneto0** [comment]
This deliberately does not copy the parse table dump to the Github pages.
The parse table dump is in an ad hoc format that I made up.  I'd rather have an actually-designed JSON version of it.

**Comment by dneto0** [comment]
It looks like the parse table generation / validation ran in the wgsl-grammar subjob
https://github.com/gpuweb/gpuweb/runs/6838654637?check_suite_focus=true#step:3:109

File: wgsl/Makefile [hunk 1]
```diff
@@ -1,24 +1,29 @@
 .PHONY: all clean
 
-all: index.html grammar/grammar.js test
+all: index.html grammar/grammar.js test wgsl.lalr.txt
```
Comment (kainino0x) [inline wgsl/Makefile:3]: note: If you want to enable this on CI without enabling it for a default `make` invocation, you could add it to `compile.sh` instead

**Review by kainino0x — APPROVED** [review]
makefile changes LGTM

> It looks like the parse table generation / validation ran in the wgsl-grammar subjob

fyi, this comes from `.gpuweb/workflows/publish-specs.yml`, where `wgsl-grammar` invokes `compile.sh`. (I think we may have some existing redundancy, as it also runs in `publish-static.yml`.)

File: wgsl/Makefile (hunk 1 above)
Comment (dneto0) [reply wgsl/Makefile:3]: Yeah, thanks.
I'll see if this bothers me or others.  Currently, the default target in wgsl also builds treesitter's WGSL grammar and checks examples. That already takes a few seconds, so I'm accustomed to manually running "make index.html" anyway.

**Pull request merged**